pkg/oc/admin/diagnostics/diagnostics/cluster/aggregated_logging/diagnostic.go

```diff
@@ -153,20 +150,30 @@ func (d *AggregatedLogging) Requirements() (client bool, host bool) {
 }
 
 func (d *AggregatedLogging) Complete(logger *log.Logger) error {
-	if len(d.MasterConfigFile) > 0 {
-		var err error
-		d.masterConfig, err = hostdiag.GetMasterConfig(d.MasterConfigFile, logger)
+	if len(d.Project) > 0 {
+		return nil
+	}
+
+	// Check if any of the default logging projects are present in the cluster
+	for _, project := range defaultLoggingProjects {
+		d.Debug("AGL0030", fmt.Sprintf("Trying default logging project %q", project))
```

Comment:
Earlier diag code `AGL0030` is used for project node selector warning. Message for the diag code is meant to be unique and by changing it, it will be hard to determine which msg we are referring to unless we know the release version/commit.
So for existing errors/warnings, keep the same diag code and for the new ones that you are introducing, pick a diag code what was not used before.